packages/govern-subgraph/schema.graphql

```diff
@@ -1,22 +1,55 @@
-type Govern @entity {
+type ERC3000Registry @entity {
```

Comment:
I included the `ERC3000Registry` to keep track of every pair (dao, queue) registered under `name`.

Reply:
Note that names are unique, and you cannot have 2 DAOs with the same name. What you can have though, is DAOs registered multiple times, with multiple names. I'm expecting this to change per Facu's review though. 😄

Reply:
Right, I used the`name` as the ID for each Entry entity (Govern, Queue) pair.